mypyc/ops.py

```diff
@@ -544,6 +567,41 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_int(self)
 
 
+class ModuleGet(RegisterOp):
+    """dest = name
```

Comment:
The docstring doesn't seem right.

Style nit: Don't have trailing `"""` on a separate line for single-line docstrings.